bundle-workflow/python/paths/script_finder.py

```diff
@@ -7,16 +7,22 @@
 @dataclass
 class ScriptFinder:
     '''
-    ScriptFinder is a helper that abstracts away the details of where to look for build and test scripts.
-    Given a component name and a checked-out Git repository, it will look in the following locations, in order, for either "build.sh" or "integtest.sh", depending on the use case:
+    ScriptFinder is a helper that abstracts away the details of where to look for build, test and install scripts.
+
+    For build.sh and integtest.sh scripts, given a component name and a checked-out Git repository, 
+    it will look in the following locations, in order:
       * Root of the Git repository
       * /scripts/<script-name> in the Git repository
       * <component_scripts_path>/<component_name>/<script-name>
       * <default_scripts_path>/<script-name>
+
+    For install.sh scripts, given a component name, it will look in the following locations, in order:
+      * <component_scripts_path>/<component_name>/<script-name>
+      * <default_scripts_path>/<script-name>
```

Comment:
Why are we not looking in the Git repository for install scripts? Seems like install logic should be owned by the plugins, same as the build and test logic.

I guess if https://github.com/opensearch-project/opensearch-plugins/issues/75 gets implemented soon then we won't need custom install scripts for long. Consider this non-blocking.

Reply:
This happens in the assembly phase where we no longer have source code checked out, we're building off output from a build. That output could contain the custom scripts, but that really is saying that that output should just install itself without custom scripts, which is either https://github.com/opensearch-project/opensearch-plugins/issues/75 or https://github.com/opensearch-project/k-NN/issues/80

Reply:
Ah, OK. Makes sense. Let's get #75 done so we can get rid of this workaround.